build_tools/azure/cpython_free_threaded_requirements.txt

```diff
@@ -0,0 +1,16 @@
+# To generate cpython_free_threaded_lock.txt, use the following command:
+# docker run -v $PWD:/scikit-learn -it ubuntu bash -c 'export DEBIAN_FRONTEND=noninteractive; apt-get -yq update; apt-get install software-properties-common ccache -y; add-apt-repository --yes ppa:deadsnakes/nightly; apt-get update -y; apt-get install -y --no-install-recommends python3.13-dev python3.13-venv python3.13-nogil; python3.13t -m venv /venvs/myenv; source /venvs/myenv/bin/activate; pip install pip-tools; pip-compile --allow-unsafe /scikit-learn/build_tools/azure/cpython_free_threaded_requirements.txt -o /scikit-learn/build_tools/azure/cpython_free_threaded_lock.txt'
```

Comment:
I confirm that it generates the same lock file as the one in this PR (I deleted my local lock file to see if it could recreate it from scratch independently).

Reply:
Apparently `--platform` is a macOS thing. On my Linux machine it says:
```
unknown flag: --plaftorm
```

Reply:
Too bad then. Let's keep the comment unchanged then.